r/tests/testthat/test-dataset-csv.R

```diff
@@ -247,7 +247,7 @@ test_that("readr parse options", {
       format = "text",
       quo = "\"",
     ),
-    "Ambiguous"
```

Comment:
We previously checked for ambiguous options, but why? Is it not sufficient to just say these short-hand ones are unrecognized? cc @nealrichardson @ianmcook

Reply:
Though it's bad practice, it's normal in R to be able to refer to function parameters using a shortened version, so this is a special case where we don't allow this if it's unclear whether the shortened version is referring to an Arrow or a readr option.

Reply:
😱  I'll add some generic handling for that in the dispatch function then.